lib/labels/github/api.ex

```diff
@@ -8,4 +8,42 @@ defmodule Labels.Github.Api do
 
   @impl true
   def get_profile(code), do: ElixirAuthGithub.github_auth(code)
+
+  @impl true
+  def get_labels(token, owner, repo) do
+    req = Req.new(base_url: "https://api.github.com")
+
+    res =
+      Req.get!(req, url: "/repos/#{owner}/#{repo}/labels?per_page=100", auth: {:bearer, token})
+
+    case res.status do
+      200 -> {:ok, res.body}
+      404 -> {:error, :not_found}
```

Comment:
If the status of the response is 404 this means the request to get the list of labels couldn't find the repository, ie either the owner or the repository name doesn't exist